drivers/flash/flash_shell.c

```diff
@@ -222,6 +224,10 @@ static int cmd_test(const struct shell *shell, size_t argc, char *argv[])
 		return -EINVAL;
 	}
 
+	if (repeat == 0) {
+		repeat = 1;
+	}
+
```

Comment:
This should have separate commit as it does not fix what the commit describes.

Reply:
It's now in a separate commit with explanation.